core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/Parameter.java

```diff
@@ -86,7 +86,7 @@ public String getGroup() {
 	/**
 	 * @return the choice
 	 */
-	public String getChoice() {
+	public String[] getChoice() {
```

Comment:
the catalog is  the same, it was already using an enum array in 2.15.1
for instance:
"pattern": { "kind": "attribute", "required": "false", "type": "string", "javaType": "org.apache.camel.ExchangePattern", "enum": [ "InOnly", "InOptionalOut", "InOut", "OutIn", "OutOnly", "OutOptionalIn", "RobustInOnly", "RobustOutOnly" ], "deprecated": "false", "description": "Sets the optional ExchangePattern used to invoke this endpoint" }